### ISSUE TITLE
windows CI error on torchscript saving -- SENet

### ISSUE THREAD

**Issue by wyli** [Project-MONAI/MONAI#1260]
**Describe the bug**
the CI error was
```
test_script_0 (tests.test_senet.TestSENET) ... 
{'dimensions': 3, 'in_channels': 1, 'out_channels': 2, 'features': (16, 20, 21, 22, 23, 11), 'upsample': 'pixelshuffle'}
Error: Process completed with exit code 1.
```
looks like it's related to concurrent pretrained model file access on Windows?

cc @IsaacYangSLA @YuanTingHsieh @behxyz @yanchengnv

additional context:
https://github.com/Project-MONAI/MONAI/runs/1418698917?check_suite_focus=true#step:9:6775

**Comment by IsaacYangSLA** [comment]
One thing to notice.  This line `{'dimensions': 3, 'in_channels': 1, 'out_channels': 2, 'features': (16, 20, 21, 22, 23, 11), 'upsample': 'pixelshuffle'}` seemed from a test in basic_unet, which got buffered and not sync'ed with other outputs because it was from a `print` statement.

**Comment by YuanTingHsieh** [comment]
maybe this will be fixed in #1268 ?

**Comment by wyli** [comment]
this is fixed by #1268 (removing the global instances)
